pkg/monitor/operator.go

```diff
@@ -76,11 +37,10 @@ func startClusterOperatorMonitoring(ctx context.Context, m Recorder, client conf
 			var conditions []monitorapi.Condition
 			for i := range co.Status.Conditions {
 				c := &co.Status.Conditions[i]
-				previous := findOperatorStatusCondition(oldCO.Status.Conditions, c.Type)
-				if previous == nil {
-					continue
-				}
-				if c.Status != previous.Status {
+				previousCondition := findOperatorStatusCondition(oldCO.Status.Conditions, c.Type)
+				// If we don't have a previous state, then we should always mark the starting state with an event.
+				// We recently had a PR that caused the kube-apiserver operator be permanently degraded and it didn't show up.
+				if previousCondition == nil || c.Status != previousCondition.Status {
```

Comment:
do we want to log here that previous condition was missing so when we go through the log we have more context?

Reply:
actually, we only log what we changed to, we don't log what we changed from. so this is in keeping with the current behavior.

Reply:
> actually, we only log what we changed to, we don't log what we changed from. so this is in keeping with the current behavior.

correct